Hunting/Assets/Scripts/GameManager/SunController.cs

```diff
@@ -10,34 +10,94 @@ public class SunController : MonoBehaviour
     private float dayTime = 60;
 
     [SerializeField]
-    private float dimFactor = 3;
+    private float currentTimeOfDay = 0.5f;
+
+    public float secondsInFullDay = 120f;
+    public float timeMultiplier = 0.5f;
+
+    public MeshRenderer cloud;

```

Comment:
A class' fields should not be directly modifiable (and good to not be directly accessible) by another class; otherwise, information-hiding is not being practiced.
What should be done is to declare them as private variables, and use the [SerializeField] attribute on them.